src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+public class Book {
+    private long id;
+    private String title;
+    private int price;
+
+    private Book(long id, String title, int price) {
+        this.id = id;
+        this.title = title;
+        this.price = price;
+    }
+
+    private Book(String title, int price) {
+        this.title = title;
+        this.price = price;
+    }
+
+    public static Book of(long id, String title, int price) {
+        return new Book(id, title, price);
+    }
+
+    public static Book of(String title, int price) {
+        return new Book(title, price);
```

Comment:
According to the checklist, when converting a ResultSet to a Book object, it's better to create an object using setters or a constructor but not both. Since you have provided factory methods, consider removing the setters and making the fields final to ensure immutability of the Book objects once created.